app/views/stats/subscriptions.html.erb

```diff
@@ -4,4 +4,28 @@
 </div>
 <div class="col-md-8 col-md-offset-1">
     <h2>Subscriptions</h2>
-</div>
+ 	<table class="table inline-grid">
+		<tr>
+			<thead>
+				<th>Subscriptions</th>
+				<th>Tags</th>
+			</thead>
+		</tr>
+		<% @tags.each do |map| %>
+		<% map.each do |subscriptions,tags| %>
```

Comment:
Here we wouldn't have to unpack things, we could skip this line!

Reply:
Okay I'll just use the tags map as it is.